app-android/src/main/java/io/github/droidkaigi/confsched2023/KaigiApp.kt

```diff
@@ -143,9 +143,9 @@ private fun NavGraphBuilder.mainScreen(
 ) {
     mainScreen(
         mainNestedGraphStateHolder = KaigiAppMainNestedGraphStateHolder(),
-        mainNestedGraph = { mainNestedNavController, padding ->
+        mainNestedGraph = { mainNestedNavController, _ ->
             nestedSessionScreens(
-                modifier = Modifier.padding(padding),
+                modifier = Modifier,
```

Comment:
Using the modifier in TimetableScreen created unwanted space at the bottom, as shown in the screenshot below.
This was caused by padding in this area, and when the padding was removed, the unwanted space disappeared.

If this padding is something that should not be removed, please comment. 🙏 

<img src="https://github.com/DroidKaigi/conference-app-2023/assets/33022494/3dcbf7a0-a205-4734-b4c0-9a2e1eb0f564" width=300 />

Reply:
I think it should be removed🙏